### ISSUE TITLE
[Q] step slider for history data

### ISSUE THREAD

**Issue by eldarkurtic** [wandb/client#3378]
Hi,
Is it possible to create a step slider when working with `history` data? The example here https://docs.wandb.ai/ref/app/features/custom-charts#how-to-show-a-step-slider-in-a-custom-chart shows this feature for `historyTable` only, and it seems that the same procedure doesn't work for `history` data.  This is my use-case: I'm running a hyperparam sweep and logging data via `wandb.keras.WandbCallback()`. Now, I would like to create a custom chart with `loss` and `val_loss` (as X and Y axes). I do this by fetching the data with this query:
<img width="491" alt="Screenshot 2022-03-15 at 17 20 29" src="https://user-images.githubusercontent.com/8884008/158423943-3dce2f51-3f7e-4ebe-917e-2d185951556b.png">
This query creates a scatter plot of the `loss`-`val_loss`, but with all epochs dumped in the same graph. 
Now, I would like to split that by adding a third dimension (i.e. `_step` or `epoch` dimension) and get the epoch slider so I can have a chart that shows `loss` - `val_loss` graph for each epoch/step separately.

**Comment by nate-wandb** [comment]
Hi @eldarkurtic, it looks like this functionality disappears when not using the `historyTable`. I can see if there is a reason for this or if this is a bug. 

This may be intended behavior but it seems like the issue is more that `wandb.keras.WandbCallback()` is not saving a `historyTable` for these metrics. I'll look into the underlying code here and see why this is the case.

Thank you,
Nate

**Comment by eldarkurtic** [comment]
Hi @nate-wandb , thanks for the prompt reply. Please let me know when you have more information on this topic.
In general, I assume there shouldn't be issues for this type of visualization, as all the data needed is available in the `history` (`val`, `val_loss` and `_step`/`epoch`).

**Comment by nate-wandb** [comment]
Hi @eldarkurtic, I've reported this as a bug. There should be an option to select a step slider when using `history`.

As a workaround in the meantime you could use a Weave query to view your history at any particular step but this will not help you visual this is the way you were hoping.

![Screen Shot 2022-03-30 at 1 09 45 PM](https://user-images.githubusercontent.com/97066933/160913120-e83e2060-0b89-4d0f-a424-c95ed36a3659.png)

**Comment by eldarkurtic** [comment]
@nate-wandb thanks a lot for the update! For now, I will just pull down the data and do custom visualizations locally 😃
